src/main/java/org/sagebionetworks/web/client/widget/entity/renderer/ChallengeParticipantsWidget.java

```diff
@@ -1,46 +1,55 @@
 package org.sagebionetworks.web.client.widget.entity.renderer;
 
 import static org.sagebionetworks.web.client.ServiceEntryPointUtils.fixServiceEntryPoint;
+
 import java.util.Map;
+
 import org.sagebionetworks.repo.model.UserProfile;
 import org.sagebionetworks.web.client.ChallengeClientAsync;
+import org.sagebionetworks.web.client.PortalGinInjector;
 import org.sagebionetworks.web.client.utils.Callback;
 import org.sagebionetworks.web.client.widget.WidgetRendererPresenter;
+import org.sagebionetworks.web.client.widget.entity.controller.SynapseAlert;
+import org.sagebionetworks.web.client.widget.pagination.BasicPaginationWidget;
 import org.sagebionetworks.web.client.widget.pagination.PageChangeListener;
-import org.sagebionetworks.web.client.widget.pagination.countbased.BasicPaginationWidget;
 import org.sagebionetworks.web.shared.UserProfilePagedResults;
 import org.sagebionetworks.web.shared.WidgetConstants;
 import org.sagebionetworks.web.shared.WikiPageKey;
+
 import com.google.gwt.user.client.rpc.AsyncCallback;
 import com.google.gwt.user.client.ui.Widget;
 import com.google.inject.Inject;
 
-public class ChallengeParticipantsWidget implements UserListView.Presenter, WidgetRendererPresenter, PageChangeListener {
+public class ChallengeParticipantsWidget implements WidgetRendererPresenter, PageChangeListener {
 
+	public static final String NO_CHALLENGE_PARTICIPANTS_FOUND_MESSAGE = "No challenge participants found.";
 	private UserListView view;
 	private Map<String, String> descriptor;
 	private ChallengeClientAsync challengeClient;
 	private String challengeId;
 	private boolean isInTeam;
-	private Callback widgetRefreshRequired;
 	private BasicPaginationWidget paginationWidget;
+	private PortalGinInjector ginInjector;
+	private SynapseAlert synAlert;
 	public static final Long DEFAULT_PARTICIPANT_LIMIT = 50L;
 	public static final Long DEFAULT_OFFSET = 0L;
 
 	@Inject
-	public ChallengeParticipantsWidget(UserListView view, BasicPaginationWidget paginationWidget, ChallengeClientAsync synapseClient) {
+	public ChallengeParticipantsWidget(UserListView view, BasicPaginationWidget paginationWidget, ChallengeClientAsync synapseClient, SynapseAlert synAlert, PortalGinInjector ginInjector) {
```

Comment:
Both the TeamMembersWidget and this ChallengeParticipantsWidget now use the same view, UserListView